tests/snapshot/snapshot/restore.yaml

```diff
@@ -98,4 +118,40 @@ chapters:
             - stage: DONE
               type: SNAPSHOT
     retry:
-      count: 3
\ No newline at end of file
+      count: 3
+  - synopsis: Restore snapshot with rename_pattern and rename_replacement.
```

Comment:
For a future PR (unless this one needs more iterating) you should break up this test into `snapshot/restore/index.yaml` for basic scenarios `restore/rename.yaml`, etc., this way each stands alone as an example that can be reused in the appropriate documentation.